### PR TITLE
Biome ID changes

### PR DESCRIPTION
**Pull request by zimage**
I changed biome ids to match what's in the wiki and in my 1.20 world and removed biomes from the legend that don't exist but were previously mapped to a biome id that is legitimate.

### CODE REVIEW

**Comment by zimage** [comment]
My branch has been synced with bedrock-viz:master and is ready to be merged if all looks well.

File: static/bedrock_viz_legend.html [hunk 1]
```diff
@@ -146,14 +146,6 @@ <h2>Biomes</h2>
     <div class="name">Desert Lakes</div>
     <div class="color-swatch" style="background-color: #b8b270">&nbsp;</div>
 </div>
-<div class="grid-cell">
-    <div class="name">Dripstone Caves</div>
-    <div class="color-swatch" style="background-color: #716358">&nbsp;</div>
-</div>
```
Comment (geoffholden) [inline static/bedrock_viz_legend.html:152]: Dripstone Caves still exists in `bedrock_viz.xml`

**Review by geoffholden — CHANGES_REQUESTED** [review]

File: static/bedrock_viz_legend.html (hunk 1 above)
Comment (zimage) [reply static/bedrock_viz_legend.html:152]: Oops. Accidentally dropped that when reconciling with the other biome-related pull request. Fixed now.

**Review by zimage — COMMENTED** [review]
I've added Dripstone caves back.

**Pull request merged**